plugins/events-node/api-report.md

```diff
@@ -14,8 +14,8 @@ export interface EventBroker {
 }
 
 // @public (undocumented)
-export interface EventParams {
-  eventPayload: unknown;
+export interface EventParams<PayloadType = unknown> {
```

Comment:
Can we call this `TPayload`? https://github.com/backstage/backstage/blob/master/STYLE.md#naming

Reply:
Yes we can! :) I like the style guide!